static/app/gettingStartedDocs/javascript/vue.tsx

```diff
@@ -68,7 +68,7 @@ npm install --save @sentry/vue
     ),
     configurations: [
       {
-        description: <h5>V2</h5>,
+        description: <h5>Vue 3</h5>,
```

Comment:
@Lms24 This will put `Vue 3` title first, but it will show `Vue 2` installation instructions. We should reorder the whole object in array, not just the title.

Reply:
I believe this is correct as it currently is but I'm realizing that the PR description doesn't reflect the intent of my change. Sorry for that (I'll update it). 

We got a report that the headings were swapped _prior_ to this PR (in #55155), meaning we correctly showed the Vue 3 _code snippet_ first but gave it the wrong heading (v2). So since we actually want to show Vue 3 first, I only swapped the headings. 

![image](https://github.com/getsentry/sentry/assets/8420481/2e185071-7868-48f7-b914-2febf1bcbc16)

Reply:
gotcha! 👍